src/highlight/__tests__/highlight.spec.ts

```diff
@@ -79,7 +76,7 @@ describe('highlight', () => {
       attribute: 'invalid',
       hit: {
         _highlightResult: {
-          name: { value: '<em>foo</em> bar' },
+          name: { value: '<mark>foo</mark> bar' },
```

Comment:
Currently, this test (should warn when attribute is not found) fails, because I could not find a way to catch the `console.warn` from IS.js.

- the warning from IS.js is correctly shown in the example applications
- `console.warn` from the highlight component is correctly caught
- it might be related to ` __DEV__` not being set in Jest (but even setting it through `globals` does nothing)

I'm not sure how thorough we should test this or if we should test it, since this is already done in IS.js.

If you have any idea please let me know 🙏 .

Reply:
as it's tested properly in vanilla, I think your test case makes sense like this

Reply:
>it might be related to __DEV__ not being set in Jest (but even setting it through globals does nothing)

That's what we do in Autocomplete, and we spy on `console.warn`.

Reply:
I tried the approach from autocomplete (using the `toWarnDev` jest extension), but I still cannot catch the `console.warn` from IS.js.

Since it is properly tested there, I feel confident we can remove this test case. Wdyt?